icu_replace.sh

```diff
@@ -1,2 +1,2 @@
-gcc -fPIC -O2 -Wall -shared -o icu_replace.so icu_replace.c `icu-config --ldflags`
-strip icu_replace.so
\ No newline at end of file
+gcc -fPIC -O2 -Wall -shared -o icu_replace.so icu_replace.c $(pkg-config --libs icu-i18n`icu-config)
```

Comment:
`$(pkg-config --libs icu-i18n)`, no ?

Reply:
Yes. Of course....  Just for the practice I'll update and resubmit the pull request.